recipes/nginx.rb

```diff
@@ -91,7 +90,7 @@
       mode '0644'
       variables(
         http_port: port,
-        server_name: site_opts['server_name']
+        server_name: site_opts['server_alias'].empty? ? site_name : site_opts['server_alias'].first
```

Comment:
server_name is supposed to be a fqdn : http://httpd.apache.org/docs/2.2/mod/core.html#servername

If people decided to use 'somethingwhichwillnotwork' then it's a problem in their implmentation not in stacks.

site_name could be  'somethingwhichwillnotwork' though, as it's just used to provide a namespace for the attributes.